src/MaRRTPathPlanNode.py

```diff
@@ -107,20 +114,20 @@ def odometryCallback(self, odometry):
         self.carPosYaw = yaw
         #print "Estimated processing odometry callback: {0} ms".format((time.time() - start)*1000)
 
-    def carSensorsCallback(self, carSensors):
+    def carSensorsCallback(self, command):
         # rospy.loginfo("carSensorsCallback")
 
         # start = time.time()
-        self.steerAngle = carSensors.steerAngle
+        # The ackermann angle [rad]
+        self.steerAngle = math.degrees((command.theta_l + command.theta_r) / 2.0)
 
         #print "Estimated processing map callback: {0} ms".format((time.time() - start)*1000);
 
-    def mapCallback(self, map):
-        self.map = map
+    def mapCallback(self, track):
+        self.map = track.cones
```

Comment:
Lets have self.track (instead self.map)
and initialize it with received track object (not with cones directly)